android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/Presenter.kt

```diff
@@ -31,23 +33,22 @@ import org.checkerframework.checker.guieffect.qual.UIEffect
  * it becomes unclear where you should write your bussiness logic.
  */
 abstract class Presenter : ScopeProvider {
-  private val behaviorRelay = BehaviorRelay.create<PresenterEvent>()
-  private val lifecycleRelay = behaviorRelay.toSerialized()
+  private val lifecycleFlow = MutableSharedFlow<PresenterEvent>()
```

Comment:
Shouldn't this be `(1, 0, DROP_OLDEST)`? Without a buffer, `tryEmit` will always fail.

Reply:
Yes, left over from testing on StateFlow